src/update_agent/actor.rs

```diff
@@ -33,6 +33,21 @@ impl Actor for UpdateAgent {
     }
 }
 
+pub struct LastRefresh {}
+
+impl Message for LastRefresh {
+    type Result = i64;
+}
+
+impl Handler<LastRefresh> for UpdateAgent {
+    type Result = i64;
+
+    fn handle(&mut self, _msg: LastRefresh, _ctx: &mut Self::Context) -> Self::Result {
```

Comment:
Self-note: it would be interesting to check whether this is handled in a concurrent way (i.e. on the spot) even when there is a long-running `RefreshTick` already being processed (e.g. an rpm-ostree deploy taking quite some time to download the new content).

Reply:
It looks like this is _not_ handled in a concurrent way. I think it is because when we handle `RefreshTick`s, we use `actix::AsyncContext::wait()`, and so it stops processing any incoming events until the future resolves, essentially making it synchronous.
I've done some experimentation here: https://gist.github.com/kelvinfan001/14f933776c51ca6e455b5797b72531c8